test/crd.go

```diff
@@ -120,6 +125,14 @@ func ConfigurationSpec(imagePath string, options *Options) *v1alpha1.Configurati
 		spec.RevisionTemplate.Spec.Container.Env = options.EnvVars
 	}
 
+	if options.ContainerResources.Limits == nil && options.ContainerResources.Requests == nil {
+		spec.RevisionTemplate.Spec.Container.Resources = corev1.ResourceRequirements{
+			Requests: corev1.ResourceList{
+				corev1.ResourceCPU: resource.MustParse(defaultRequestCPU),
+			},
+		}
+	}
```

Comment:
I think we should apply this defaulting behavior before actually putting this into the ConfigurationSpec itself.

That'd mean: Move this block above the `spec :=` and make it default on the option itself, i.e.:

```golang
if options.ContainerResources.Limits == nil && options.ContainerResources.Requests == nil {
	options.ContainerResources = corev1.ResourceRequirements{
		Requests: corev1.ResourceList{
			corev1.ResourceCPU: resource.MustParse(defaultRequestCPU),
		},
	}
}
```